package.json

```diff
@@ -46,25 +46,25 @@
 		"test": "node ./out/test/runTest.js"
 	},
 	"devDependencies": {
-		"@types/glob": "^7.1.1",
-		"@types/mocha": "^7.0.1",
-		"@types/node": "^12.11.7",
-		"@types/vscode": "^1.42.0",
+		"@types/glob": "^7.1.3",
+		"@types/mocha": "^7.0.2",
+		"@types/node": "^12.19.16",
+		"@types/vscode": "^1.53.0",
+		"@typescript-eslint/eslint-plugin": "^2.34.0",
+		"@typescript-eslint/parser": "^2.34.0",
 		"eslint": "^6.8.0",
-		"@typescript-eslint/parser": "^2.18.0",
-		"@typescript-eslint/eslint-plugin": "^2.18.0",
 		"glob": "^7.1.6",
-		"mocha": "^7.0.1",
-		"typescript": "^3.7.5",
-		"vscode-test": "^1.3.0"
+		"mocha": "^7.2.0",
+		"typescript": "^3.9.9",
+		"vscode-test": "^1.5.0"
 	},
 	"dependencies": {
-		"adbkit": "github:mhama/adbkit",
+		"@devicefarmer/adbkit": "^2.11.3",
```

Comment:
use devicefarmers adbkit instead, it's more updated https://github.com/DeviceFarmer/adbkit